src/types/data.ts

```diff
@@ -1091,6 +1091,7 @@ export type RequestPropsMap = {
   VKWebAppHideBannerAd: {};
   VKWebAppShowBannerAd: ShowBannerAdRequest;
   VKWebAppShowNativeAds: ShowNativeAdsRequest;
+  VKWebAppGetAds: [key: string | number];
```

Comment:
Этот тип не поддерживается в Typescript < v4

Для обратной совместимости лучше просто 

```suggestion
  VKWebAppGetAds: [string | number];
```

Reply:
Хорошо. Спасибо, что написали. Буду знать!

Reply:
Обрати внимание, что упал CI https://github.com/VKCOM/vk-bridge/pull/312/checks

Мы как раз начали пакет спецом прогонять TS 3.8.3, чтобы проверить, что мы обратную совместимость по типам не ломаем (подробнее в https://github.com/VKCOM/vk-bridge/pull/340)